src/js/lib/ajax.js

```diff
@@ -101,6 +103,8 @@ var ajax = function(opt, success, failure) {
       try {
         if (opt.type === 'json') {
           body = JSON.parse(body);
+        } else if (op.type === 'xml') {
```

Comment:
Right, I forgot about that. I suppose if someone really wanted to, they could get their values from XML with regexes.